quarkus/server/build.gradle.kts

```diff
@@ -84,78 +79,13 @@ tasks.named<QuarkusRun>("quarkusRun") {
     listOf("-Dpolaris.bootstrap.credentials=POLARIS,root,secret", "-Dquarkus.console.color=true")
 }
 
-distributions {
-  main {
-    contents {
-      from(runScript)
-      from(project.layout.buildDirectory.dir("quarkus-app"))
-      from("distribution/NOTICE")
-      from("distribution/LICENSE")
-      from("distribution/README.md")
-      from("distribution/DISCLAIMER")
-    }
-  }
-}
-
 val quarkusBuild = tasks.named<QuarkusBuild>("quarkusBuild")
 
-val distTar =
-  tasks.named<Tar>("distTar") {
-    dependsOn(quarkusBuild)
-    inputs.files(runScript)
-    compression = Compression.GZIP
-  }
-
-val distZip =
-  tasks.named<Zip>("distZip") {
-    dependsOn(quarkusBuild)
-    inputs.files(runScript)
-  }
-
-val digestDistTar =
-  tasks.register<GenerateDigest>("digestDistTar") {
-    description = "Generate the distribution tar digest"
-    mustRunAfter(distTar)
-    file.set { distTar.get().archiveFile.get().asFile }
-  }
-
-val digestDistZip =
-  tasks.register<GenerateDigest>("digestDistZip") {
-    description = "Generate the distribution zip digest"
-    mustRunAfter(distZip)
-    file.set { distZip.get().archiveFile.get().asFile }
-  }
-
-distTar.configure { finalizedBy(digestDistTar) }
-
-distZip.configure { finalizedBy(digestDistZip) }
-
-if (project.hasProperty("release") || project.hasProperty("signArtifacts")) {
-  signing {
-    sign(distTar.get())
-    sign(distZip.get())
-  }
-}
-
 // Expose runnable jar via quarkusRunner configuration for integration-tests that require the
 // server.
 artifacts {
   add(quarkusRunner.name, provider { quarkusBuild.get().fastJar.resolve("quarkus-run.jar") }) {
     builtBy(quarkusBuild)
   }
-  add(distributionTar.name, provider { distTar.get().archiveFile }) { builtBy(distTar) }
-  add(distributionTar.name, provider { digestDistTar.get().outputFile }) { builtBy(digestDistTar) }
-  add(distributionZip.name, provider { distZip.get().archiveFile }) { builtBy(distZip) }
-  add(distributionZip.name, provider { digestDistZip.get().outputFile }) { builtBy(digestDistZip) }
-}
-
-afterEvaluate {
-  publishing {
-    publications {
-      named<MavenPublication>("maven") {
-        artifact(distTar.get().archiveFile) { builtBy(distTar) }
-        artifact(distZip.get().archiveFile) { builtBy(distZip) }
-      }
-    }
-  }
+  add("distributionElements", layout.buildDirectory.dir("quarkus-app")) { builtBy("quarkusBuild") }
```

Comment:
After untar:
```
$ ls server/
app  lib  logs  quarkus  quarkus-app-dependencies.txt  quarkus-run.jar
```
Is `quarkus-app-dependencies.txt` intended to be in the distro? (same for admin)

Reply:
It is noise and distraction from my POV. Not useful to end users.

Reply:
Removed in the new commit.